### ISSUE TITLE
IUPHAR commented out

### ISSUE THREAD

**Issue by PhillipsOwen** [biolink/biolink-model#743]
**Is your feature request related to a problem? Please describe.**
It would appear that IUPHAR is commented out in the latest biolink model yaml (1.8.2)

**What working group (or team) did this request originate from?**
Translator SRI

**Note:** This is relevant for members of NCATS Translator.

**Describe the solution you'd like**
Why was this commented out? This identifier is used in pharmacology data to identify gene to drug interactions.

**Additional information to support this request** (optional)
see the biolink-model.yaml file, line 6556 (or search for "# - IUPHAR")

**Tag relevant members for discussion**
@cbizon @RichardBruskiewich @cmungall

**Comment by sierra-moxon** [comment]
Hi @PhillipsOwen - It looks like this prefix was removed in 1.4.0 (also IUPHAR.FAMILY was added to 'gene or gene product', and in more recent releases, 'gene or gene product' became a mixin).  

I don't see IUPHAR as a plain prefix in identifiers.org.  Instead, it lists:
IUPHAR.LIGAND
IUPHAR.RECEPTOR
IUPHAR.FAMILY

does IUPHAR.FAMILY help your use case?

**Comment by PhillipsOwen** [comment]
TL;DR - I no longer need this as there is other data in the data set that can be leveraged for gene ids.

i was looking at a previous parsing of the GtoPDB data and noticed that nothing normalized properly. as it turns out this was due to the lack of IUPHAR data in the node normalizer. which led me to the IUPHAR being commented out in the biolink model. I interpreted this as an issue as IUPHAR curies are legit.

However, the latest GtoPDB data has HGNC id references that I can use (which are supported by the normalizer). since i have a work around i no longer have an issue.

sorry to waste your time. i am closing the issue.

**Comment by sierra-moxon** [comment]
No problem at all - awesome that you found something that works! :)
